src/communication/network_interface_sr.c

```diff
@@ -10849,13 +10849,17 @@ sflashback_get_summary (THREAD_ENTRY * thread_p, unsigned int rid, char *request
       goto error;
     }
 
+  FLASHBACK_SET_REQUEST_TIME ();
+
   /* get summary list */
   error_code = flashback_make_summary_list (thread_p, &context);
   if (error_code != NO_ERROR)
     {
       goto error;
     }
 
+  FLASHBACK_SET_MIN_LOG_PAGEID_TO_KEEP (&context.start_lsa);
```

Comment:
I think this mechanism need to consider the following things:
- context.start_lsa == NULL_LSA
- between getting summaries and getting log infos

Reply:
How about checking the value of the 'supplemental_log' and returning an error with an error_code if its value is 0 in the first place of this function?

Reply:
1. yes
   * I will add `assert(!LSA_ISNULL(&context.start_lsa))`
   * I will check intervals between requests, and if intervals exceeds threshold then return error
2. Yes, I will check supplemental_log is on at utility side, and add `assert (supplemental_log >=1)` here.